tests/outputs/a_manurev/refereeReports.yaml

```diff
@@ -1,21 +1,27 @@
 1902cl_actaArev:
   claimed_found_what:
+    -
   claimed_why_important:
+    -
   did_how:
+    -
   did_what:
-  due_date: '2020-09-15'
+    -
+  due_date: ''
```

Comment:
this should be passed in

Reply:
Hi Dr. Billinge, 

I think you are looking at the outdated commit. The new commit fixes all these and passes the tests.